bots/images/scripts/ovirt.setup

```diff
@@ -0,0 +1,151 @@
+#!/bin/bash
+
+set -eux
+
+echo foobar | passwd --stdin root
+
+/usr/bin/systemctl stop firewalld NetworkManager kdump
+/usr/bin/systemctl disable firewalld NetworkManager kdump
```

Comment:
Please avoid hardcoded paths; This very plausibly will move to `/bin/` in newer OSes. There are plenty more instances below, I don't mention them all individually.